app/graphql/mutations/report_cloud_fixity.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+class Mutations::ReportCloudFixity < Mutations::BaseMutation
+  null true
+
+  argument :preservation_object_id, ID, required: true
+  argument :file_metadata_node_id, ID, required: true
+  argument :status, String, required: true
+
+  field :resource, Types::Resource, null: false
+  field :errors, [String], null: true
+
+  def resolve(preservation_object_id, file_metadata_node_id, status)
+    preservation_object = query_service.find_by(id: preservation_object_id)
+    file_metadata_node = query_service.find_by(id: file_metadata_node_id)
```

Comment:
We don't want to actually save the FileMetadataNodes right? Still leave them as embedded objects?